src/impls.rs

```diff
@@ -72,6 +72,8 @@ safety_comment! {
     unsafe_impl!(isize: Immutable, TryFromBytes, FromZeros, FromBytes, IntoBytes);
     unsafe_impl!(f32: Immutable, TryFromBytes, FromZeros, FromBytes, IntoBytes);
     unsafe_impl!(f64: Immutable, TryFromBytes, FromZeros, FromBytes, IntoBytes);
+    unsafe_impl!(#[cfg(feature="float-nightly")] f16: Immutable, TryFromBytes, FromZeros, FromBytes, IntoBytes);
+    unsafe_impl!(#[cfg(feature="float-nightly")] f128: Immutable, TryFromBytes, FromZeros, FromBytes, IntoBytes);
```

Comment:
Following the pattern here: https://github.com/google/zerocopy/blob/9e04925a5684438e6a080116b8b4c4a43ae6a33f/src/impls.rs#L370-L374

```suggestion
    #[cfg(feature="float-nightly")] 
    unsafe_impl!(#[cfg_attr(doc_cfg, doc(cfg(feature = "float-nightly")))] f16: Immutable, TryFromBytes, FromZeros, FromBytes, IntoBytes);
    #[cfg(feature="float-nightly")] 
    unsafe_impl!(#[cfg_attr(doc_cfg, doc(cfg(feature = "float-nightly")))] f128: Immutable, TryFromBytes, FromZeros, FromBytes, IntoBytes);
```